test/gtest/ucm/malloc_hook.cc

```diff
@@ -973,25 +973,25 @@ UCS_TEST_F(malloc_hook, dlopen) {
     UCS_TEST_MESSAGE << "Loading " << lib_mmap;
 
     lib = dlopen(lib_load.c_str(), RTLD_NOW);
-    EXPECT_NE((uintptr_t)lib, NULL);
+    EXPECT_NE((uintptr_t)lib, (uintptr_t)NULL);
```

Comment:
why not just remove the casting to uintptr_t?

Reply:
without casting it fails because depends from NULL definition - void* or integer
```
error: ISO C++ forbids comparison between pointer and integer [-fpermissive]
```